packages/experience/src/components/PasswordSignInForm/index.tsx

```diff
@@ -129,7 +129,8 @@ const PasswordSignInForm = ({ className, autoFocus, signInMethods }: Props) => {
             isDanger={!!errors.identifier}
             errorMessage={errors.identifier?.message}
             enabledTypes={signInMethods}
-            defaultValue={searchParams.get(ExtraParamsKey.LoginHint) ?? undefined}
+            defaultValue={field.value.value}
```

Comment:
If you have default value set in `useForm`, do you still need to set the default value here? Maybe not

Reply:
There is no `value` for the `SmartInputField`, need to specify the initial value to the `defaultValue` prop.